DataFormats/Detectors/FIT/FT0/include/DataFormatsFT0/ChannelData.h

```diff
@@ -15,19 +15,20 @@
 #ifndef _FT0_CHANNELDATA_H_
 #define _FT0_CHANNELDATA_H_
 
+#include <Framework/Logger.h>
```

Comment:
Why did your add this line? We don't print anything in header. And similar remark about DataFormats/Detectors/FIT/FT0/include/DataFormatsFT0/Digit.h

Reply:
@AllaMaevskaya check srcs

Reply:
so move include in srcs

Reply:
Ok